warehouse/static/sass/blocks/_notification-bar.scss

```diff
@@ -33,6 +33,30 @@
     - dismissable: Indicates notification be be dismissed. Defaults to hidden.
     - visible: Indicates a visible, non-dismissed notification
 */
+/*
+  A notification bar can be made dismissable by adding the following to the main
```

Comment:
I think the placement of this is fine, however, I think you can include it in the same comment block :)